prowler/lib/mutelist/mutelist.py

```diff
@@ -317,12 +321,19 @@ def is_item_matched(matched_items, finding_items):
         try:
             is_item_matched = False
             if matched_items and (finding_items or finding_items == ""):
+                if tag:
+                    is_item_matched = True
                 for item in matched_items:
                     if item.startswith("*"):
                         item = ".*" + item[1:]
-                    if re.search(item, finding_items):
-                        is_item_matched = True
-                        break
+                    if tag:
+                        if not re.search(item, finding_items):
+                            is_item_matched = False
+                            break
```

Comment:
Why do you need this? Out of curiosity

Reply:
If I find a tag that it's not inside finding_items, it won't match the item because AND logic.

Reply:
Understood, thanks for clarifying it!